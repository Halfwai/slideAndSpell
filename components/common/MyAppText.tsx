import React from 'react';
import{Text, TextStyle, StyleSheet} from 'react-native';

interface MyAppTextProps {
    children: React.ReactNode;
    style?: TextStyle;
}

export default function MyAppText(props: MyAppTextProps) {
    return <Text style={[styles.text, props.style]}>{props.children}</Text>
}

const styles = StyleSheet.create({
    text: {
        fontFamily: 'Quicksand'
    }
});