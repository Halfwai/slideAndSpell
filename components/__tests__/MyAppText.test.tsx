import { render } from '@testing-library/react-native';
import MyAppText from '@/components/common/MyAppText';

it(`MyAppText Component renders correctly`, () => {
    const { toJSON } = render(
        <MyAppText>Test</MyAppText>
    );    
    expect(toJSON()).toMatchSnapshot();
});