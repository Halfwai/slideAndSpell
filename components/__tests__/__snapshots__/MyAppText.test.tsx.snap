// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyAppText Component renders correctly 1`] = `
<Text
  style={
    [
      {
        "fontFamily": "Quicksand",
      },
      undefined,
    ]
  }
>
  Test
</Text>
`;
